test/accordion.test.js

```diff
@@ -0,0 +1,106 @@
+import {
+	assert, html, fixture
+} from '@open-wc/testing';
+import { tap } from '@polymer/iron-test-helpers/mock-interactions.js';
+
+suite('basic', () => {
+	let tabs;
+
+	suiteSetup(async () => {
+		await import('../cosmoz-tabs.js');
+	});
+
+	setup(async () => {
+		tabs = await fixture(html`
+			<cosmoz-tabs accordion selected-values='["tab1","tab2"]'>
+				<cosmoz-tab name="tab0">Tab text 1</cosmoz-tab>
+				<cosmoz-tab name="tab1">Tab text 2</cosmoz-tab>
+				<cosmoz-tab name="tab2">Tab text 3</cosmoz-tab>
+			</cosmoz-tabs>
+		`);
+		const onIronItemsChanged = () => {
```

Comment:
We should use `oneEvent` from '@open-wc/testing' here.

Reply:
Why are we even registering these event listeneres everywhere?